amplicon-pooling.ipynb

```diff
@@ -817,11 +818,11 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/Users/yoshikivazquezbaeza/git_sw/metagenomics_pooling_notebook/metapool/sample_sheet.py:394: UserWarning: The column I5_Index_ID in the sample sheet is empty\n",
+      "/Users/yoshikivazquezbaeza/git_sw/metagenomics_pooling_notebook/metapool/sample_sheet.py:406: UserWarning: The column I5_Index_ID in the sample sheet is empty\n",
```

Comment:
@ElDeveloper Perhaps your home directory could be replaced with something more generalized. Maybe an environment variable?

Reply:
Unfortunately, I don't have control over that. This is likely not clear from the diff but this path is the text that shows up from showing a warning i.e. from calling:

```python
import warnings
warnings.warn('<- That's going to be a path to the file where the this warning came from')
```

Reply:
Ah I see. Then it's a non-issue.